internal/ansible/ansible_manager.go

```diff
@@ -0,0 +1,351 @@
+package ansible
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"sync"
+	"time"
+
+	"git.sr.ht/~spc/go-log"
+	"github.com/apenella/go-ansible/pkg/execute"
+	"github.com/apenella/go-ansible/pkg/options"
+	"github.com/apenella/go-ansible/pkg/playbook"
+	ansibleResults "github.com/apenella/go-ansible/pkg/stdoutcallback/results"
+	"github.com/hashicorp/go-multierror"
+	"github.com/project-flotta/flotta-device-worker/internal/ansible/dispatcher"
+	"github.com/project-flotta/flotta-device-worker/internal/ansible/mapping"
+	"github.com/project-flotta/flotta-device-worker/internal/ansible/model/message"
+	"github.com/project-flotta/flotta-operator/models"
+
+	pb "github.com/redhatinsights/yggdrasil/protocol"
+)
+
+// AnsibleManager handle ansible playbook execution
+type AnsibleManager struct {
+	wg                *sync.WaitGroup
```

Comment:
why a pointer here, maybe is not needed at all and we can avoid null exceptions, no?